### ISSUE TITLE
`colors.webpage.bg` not respected when killing renderer process

### ISSUE THREAD

**Issue by leo848** [qutebrowser/qutebrowser#7484]
**Version info**:
https://paste.the-compiler.org/view/47e89671

**Does the bug happen if you start with `--temp-basedir`?**:
Yes

**Description**
When the renderer process for a specific tab is killed and the tab is reloaded, `colors.webpage.bg` is not respected anymore.

**How to reproduce**

1. `qutebrowser --temp-basedir --set "colors.webpage.bg" "#5865f2" about://blank`
2. Notice that the page has a [blurple](https://en.wikipedia.org/wiki/Shades_of_blue#Blurple_(2015-2021)) color, as expected.
3. Kill the renderer process. I don't know a programmatic way to do this but it's easy via `htop` or one of the many alternatives.
4. The "Unable to load page" screen should appear. Click on the "Try again" button.
5. This tab now permanently has a white page background. Reloading the tab doesn't fix it, but closing and re-opening does (or just opening a new `about://blank` tab).

This issue is of course not very important (i mean, how often do you guys kill your rendering processes :smile: ), but still something I noticed in my normal usage.

**Comment by The-Compiler** [comment]
Reproducer:

```
qutebrowser --temp-basedir --set "colors.webpage.bg" "#5865f2" about://blank ":later 1000 open chrome://kill" ":later 2000 open about:blank"
```

also reproduces with Qt 6.4. Setting it after the crash fixes things:

```
qutebrowser --temp-basedir --set "colors.webpage.bg" "#5865f2" about://blank ":later 1000 open chrome://kill" ":later 1500 set colors.webpage.bg #500050" ":later 2000 open about:blank"
```

Seems like an upstream bug to me, but I suppose qutebrowser could just apply the setting again after a renderer process crash.

**Comment by rudism** [comment]
I find myself having to set `colors.webpage.bg` back to `white` multiple times during single browsing sessions to make pages readable, would that be due to this upstream bug? Does the rendering process regularly crash and restart itself without any kind of visible indication in the browser itself?

**Comment by The-Compiler** [comment]
No, if the renderer process crashes, you should get a messag about it (you can `:open chrome://crash` to get that).
